Uplift/Views/Cells/FacilitiesDropdownCell.swift

```diff
@@ -150,33 +110,63 @@ extension FacilitiesDropdownCell: UICollectionViewDataSource {
     }
 
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        // TODO: once facility cells are finished
-        return UICollectionViewCell(frame: .zero)
+        switch facility.details[indexPath.row].detailType {
+        case .equipment:
+            //swiftlint:disable:next force_cast
```

Comment:
maybe we should just remove this from swiftlint as a rule so we dont have to do this everywhere. thoughts? @ywsang @CameronHamidi  @pnor

Reply:
I think we can keep it, we only use it when we dequeue cells and it's a good reminder of proper linting practice.